src/components/common/Loading.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
If this is meant to be reusable I don't think a registering prop makes sense, not everywhere we shall use a loader we'll be registering

Reply:
Thanks for the feedback this has been worked on.